### PR TITLE
Enables searching on staff list

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13031

## Testing
**URL to test:** [Netlify](https://deploy-preview-1084--atd-moped-main.netlify.app/moped/staff/)

**Steps to test:**
1. Go to the staff page.
2. Use the search input at the top-right corner of the staff list to search for a staff member. 

🎉 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/staff/StaffListView.js [hunk 1]
```diff
@@ -106,6 +105,7 @@ const StaffListView = () => {
               columns={staffColumns}
               getRowId={(row) => row.user_id}
               slots={{ toolbar: GridToolbar }}
+              slotProps={{ toolbar: { showQuickFilter: true } }}
```
Comment (johnclary) [inline moped-editor/src/views/staff/StaffListView.js:108]: this is the only change :)

**Review by frankhereford — APPROVED** [review]
Tested it on netlify - looks great.

**Review by chiaberry — APPROVED** [review]
🔎 ✔️

**Review by mddilley — APPROVED** [review]
Looks leveled up to me! 🍄 🙏 Awesome to see more payoff from the DataGrid update ✨

**Review by roseeichelmann — APPROVED** [review]
looks and works great :sparkles:

**Comment by johnclary** [comment]
thanks!

**Pull request merged**